src/workerd/tests/bench-response.c++

```diff
@@ -88,9 +88,12 @@ BENCHMARK_F(Response, arrayBufferBody)(benchmark::State& state) {
 BENCHMARK_F(Response, jsonResponse)(benchmark::State& state) {
   fixture->runInIoContext([&](const TestFixture::Environment& env) {
     auto& js = env.js;
+    // Prepare object to serialize. Do this outside the loop to avoid measuring its repeated
+    // construction cost. What we want to measure is just the cost of the api::Response::json_ call.
+    kj::StringPtr keys[] = {"key"_kj};
+    jsg::JsValue values[] = {env.js.str("value"_kj)};
+    auto obj = js.obj(kj::ArrayPtr(keys), kj::ArrayPtr(values));
```

Comment:
Apparently, objects created with this does not have fast properties. We should revert the changes on main.